kbase-extension/static/kbase/js/common/jobManager.js

```diff
@@ -10,14 +10,8 @@ define([
     const validOutgoingMessageTypes = Object.values(jcm.RESPONSES);
 
     const jobCommand = {
-        cancel: {
-            command: jcm.MESSAGE_TYPE.CANCEL,
-            listener: jcm.MESSAGE_TYPE.STATUS,
-        },
-        retry: {
-            command: jcm.MESSAGE_TYPE.RETRY,
-            listener: jcm.MESSAGE_TYPE.RETRY,
-        },
```

Comment:
Major change in this file is having the job manager set up all relevant job message listeners when it loads the data for a batch job. Adding these extra listeners is therefore no longer necessary.